bz-manage.sh

```diff
@@ -381,4 +382,137 @@ get_bz_id_summary() {
   done
 }
 
+update_doc_text() {
+  local queryurl jq bz bz_assignee doc_type doc_text gh_pr gh_pr_info gh_pr_title gh_pr_desc gh_pr_type update_bz
+  query_file=.query_file.json
+
+  # If a BZ has required_doc_text unset and GH PR set:
+  # Automatically add doc text from PR description
+  queryurl="${BASE_URL}"
+  queryurl+="&f1=flagtypes.name&f2=external_bugzilla.url&f3=ext_bz_bug_map.ext_bz_bug_id"
+  # substring doesn't work with ext_bz_bug_map.ext_bz_bug_id, using anywordssubstr
+  queryurl+="&o1=notsubstring&o2=substring&o3=anywordssubstr"
```

Comment:
Bugzilla does not search for links properly for some reason, I found a workaround with using anywordssubstr filter instead of substring. Apparently, this works. And for `not anywordssubstr` I set also n=1 in some below search.

Reply:
ok - works for me